pkgs/top-level/all-packages.nix

```diff
@@ -26432,10 +26440,12 @@ in
 
   wayfireApplications = wayfireApplications-unwrapped.withPlugins (plugins: [ plugins.wf-shell ]);
   inherit (wayfireApplications) wayfire wcm;
-  wayfireApplications-unwrapped = recurseIntoAttrs (callPackage ../applications/window-managers/wayfire/applications.nix { });
-  wayfirePlugins = recurseIntoAttrs (callPackage ../applications/window-managers/wayfire/plugins.nix {
+  wayfireApplications-unwrapped =
+    (callPackage ../applications/window-managers/wayfire/applications.nix { }).
+    extend (_: _: { wlroots = wlroots_0_12; });
+  wayfirePlugins = callPackage ../applications/window-managers/wayfire/plugins.nix {
```

Comment:
We should re-add `recurseIntoAttrs` for `wayfireApplications-unwrapped` and `wayfirePlugins` (relevant for nix-env, the package search, etc.). That should also fix this nixpkgs-review output:
```
3 packages removed:
wayfire (†0.7.0) wcm (†0.7.0) wf-shell (†0.7.0)
```

Reply:
If you don't mind you could also drop `being cage and wayfire.` from this commit message (there are more affected packages now but we don't need to name them in the commit message).

(The Wayfire change could be an extra commit as we're also making it extensible but that's likely not worth the effort so feel free to ignore that, depending on what you prefer.)

Reply:
@primeos I have pushed something that compiles, checking the result of nixpkgs-review now

Reply:
Hmm, now it still says 

```
1 package removed:
wf-shell (†0.7.0)
```

Not sure why...

Reply:
It still missing for `wayfirePlugins` (will appear as `wayfireApplications-unwrapped.wayfirePlugins.wf-shell` in the search). Then it should work.

Reply:
Oh, I see... Should it be on both or only on unwrapped?

Reply:
It used to be on both `wayfireApplications-unwrapped` and `wayfirePlugins` (s. diff) which is why I'd keep it on both.

Reply:
Okay. Marking this resolved 👍🏻